app/templates/build/config.hbs

```diff
@@ -1,4 +1,5 @@
 <Request::Configs
+  @repo={{this.model.repo}}
```

Comment:
It seems like this value can be just derived from `@request`, isn't it?

Reply:
i believe i thought i'd need to add it because now that same view is also shared on the `repo` route (see below), where there is no request.

Reply:
Yeah, I've noticed that as well. Internally in component it could be an alias:

```js
export default Component.extend({
  request: null,
  repo: reads('request.repo')
});
```
And then you can overwrite it on invocation:
```hbs
<Request::Configs
  @repo={{this.repo}}
>
```
With this invocation, external attribute will take precedence over internal property and overwrite it.

Reply:
oh, i see ... yes, that makes sense.

Reply:
lol, somehow that's already been taken care of in the final branch. see https://github.com/travis-ci/travis-web/blob/sf-request-config-tweaks/app/templates/build/config.hbs and https://github.com/travis-ci/travis-web/blob/sf-request-config-tweaks/app/templates/job/config.hbs